morphia/src/main/java/org/mongodb/morphia/query/Query.java

```diff
@@ -15,166 +15,142 @@
  * @author Scott Hernandez
```

Comment:
This is a mess.  What does this have to do with the reference documentation?  And why are the diffs so difficult to parse?  Did you move all the methods around?